README.md

```diff
@@ -39,46 +40,62 @@ Concretely, it's a *typed* content address: a tuple of `(content-type, content-a
 
 Current version: CIDv1
 
-A CIDv1 has four parts:
+The atomic CIDv1 is a **binary** format composed from `unsigned_varint`s that encode values from the , each of which :
```

Comment:
what does the word "atomic" buy us here?

Reply:
I'll also take issue with `unsigned_varint` - both its code quoting, and also the `_`; I think this would probably do the job:

```suggestion
The atomic CIDv1 is a **binary** format composed from [unsigned varints](https://github.com/multiformats/unsigned-varint) that encode values from the , each of which :
```

Reply:
Actually there's a few more problems in this sentence, here's my proposed version:

```suggestion
CIDv1 is a **binary** format composed of [unsigned varints](https://github.com/multiformats/unsigned-varint) prefixing a hash digest to form a self-describing "content address":
```